### PR TITLE
[fr] Follows #5106, #5123 & #5130

### PR DESCRIPTION
**Pull request by HavokInspiration**
Follows #5106, #5123 & #5130

I posted a comment on #5130 because I think the wrong class might have been included. See https://github.com/cakephp/docs/pull/5130#issuecomment-318842069

I'll be glad revert it back and fix if I'm mistaken :smile: 

(I took care of the JA version for #5130)

### CODE REVIEW

**Comment by HavokInspiration** [comment]
I added a commit that follows #5106

File: fr/appendices/3-5-migration-guide.rst [hunk 1]
```diff
@@ -165,6 +165,15 @@ le moteur était mal configuré ou indisponible. Reportez-vous à la section
 :ref:`cache-configuration-fallback` pour plus d'informations sur la configuration
 de 'fallbacks' pour vos configurations de cache.
 
+Console Integration Testing
+---------------------------
+
+La classe ``Cake\TestSuite\ConsoleIntegrationTestCase`` a été ajoutée pour
+faciliter les tests d'intégrations des applications console. Pour plus d'informations
```
Comment (antograssiot) [inline fr/appendices/3-5-migration-guide.rst]: Pas sur pour le s à integration....

**Comment by HavokInspiration** [comment]
@antograssiot Corrigé. Il n'y a pas de S aux autres occurrences de cette expression dans le reste de la doc donc autant uniformiser.

**Comment by antograssiot** [comment]
:clap:

**Pull request merged**